### PR TITLE
Fix WebRequestAuthRequredEvent functions

### PR DESCRIPTION
**Pull request by NN---**

### CODE REVIEW

File: chrome/chrome.d.ts [hunk 1]
```diff
@@ -2422,8 +2422,8 @@ declare module chrome.webRequest {
     }
 
     interface WebRequestAuthRequiredEvent extends chrome.events.Event { 
-        addListener(callback: (details: OnAuthRequiredDetails, callback?: (response: BlockingResponse) => void, filter?: RequestFilter, opt_extraInfoSpec?: string[]) => void): void;
-        removeListener(callback: (details: OnAuthRequiredDetails, callback?: (response: BlockingResponse) => void) => void): void;
+        addListener(callback: (details: OnAuthRequiredDetails) => BlockingResponse, filter?: RequestFilter, opt_extraInfoSpec?: string[]): void;
+        removeListener(callback: (details: OnAuthRequiredDetails) => BlockingResponse) : void;
```
Comment (vvakame) [inline chrome/chrome.d.ts]: is this right?
https://developer.chrome.com/extensions/webRequest#event-onAuthRequired
I seem to be old definitions is right.

**Comment by NN---** [comment]
Oh , right. My bad..

File: chrome/chrome.d.ts (hunk 1 above)
Comment (NN---) [inline chrome/chrome.d.ts]: You are correct.
I wrote the correct callback type now.
The bug was that filter and opt_extraInfoSpec where part of callback and not part of addListener.

According to this: https://developer.chrome.com/extensions/webRequest#event-onAuthRequired

**Comment by NN---** [comment]
Can you merge this pull request ? Thanks.

**Comment by vvakame** [comment]
@NN--- thanks mate!

**Pull request merged**